windows/bootstrap.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+QTOX_DIR=`pwd`/..
+
+if [ -d $QTOX_DIR/libs ]; then
+    echo "Remove ./libs and redownload/recompile dependencies?"
+    read -p "m/a/N (missing/all/no): " yn
+    yn=$(echo $yn | tr "[:upper:]" "[:lower:]")
+    if [ "$yn" == "a" ]; then
+        rm -rf $QTOX_DIR/libs
+    elif [ "$yn" == "n" ]; then
+        exit -1
+    elif [ "$yn" != "m" ]; then
+        exit -1
+    fi
+fi
+
+mkdir -p $QTOX_DIR/libs
+cd $QTOX_DIR/libs
+
+
+## toxcore
+if [ ! -f "libtoxcore-win32-i686.zip" ]; then
+   wget --no-check-certificate http://jenkins.libtoxcore.so/job/libtoxcore-win32-i686/lastSuccessfulBuild/artifact/libtoxcore-win32-i686.zip
```

Comment:
Not checking certificates is something that should be done with care, https is still useful, despite the flaws in the certificate system. (I assume it's related to running a Unix utility on Windows that SSL/TLS becomes Difficult™?)